entries/jQuery.widget.xml

```diff
@@ -200,6 +215,43 @@
 			<xi:include href="../includes/widget-method-instance.xml" xmlns:xi="http://www.w3.org/2003/XInclude"/>
 			<xi:include href="../includes/widget-method-option.xml" xmlns:xi="http://www.w3.org/2003/XInclude"/>
 			<xi:include href="../includes/widget-method-widget.xml" xmlns:xi="http://www.w3.org/2003/XInclude"/>
+			<method name="_addClass">
+				<desc>
+					Add classes to an element of the widget.
+					<p>This provides a hook for the user to add additional classes or replace default styling classes, through the <a href="#option-classes"><code>classes</code> option</a>.</p>
+					<p>It also provides automatic removal of these classes when a widget is destroyed.</p>
+				</desc>
+				<argument name="element" type="jQuery" optional="true">
+					<desc>The element to add the classes to. Defaults to <code>this.element</code>.</desc>
+				</argument>
+				<argument name="keys" type="String">
+					<desc>
+						The classes to add. If a property of the <code>classes</code> option matches a key, the value will be added as well.
+						<p>Like the <code>element</code> argument, this argument is also optional. To skip it, specify <code>null</code>.</p>
+					</desc>
+				</argument>
+				<argument name="extra" type="String" optional="true">
+					<desc>Additional classes to add. Unlike the <code>keys</code> argument, these aren't associated with any properties of the <code>classes</code> option. Just like <code>keys</code>, they will also be automatically removed when destroying the widget.</desc>
+				</argument>
+				<example>
+					<desc>Add the "loading-indicator" class to the widget's element (<code>this.element</code>. Will also add any additional classes specified through the <code>classes</code> option for the given class.</desc>
+					<code><![CDATA[
+this._addClass( "loading-indicator" );
+]]></code>
```

Comment:
This contradicts the earlier sentence 'In other words, any class prefixed with namespace and widget, like "ui-accordion-"...' and it isn't a very realistic case for widgets that follow our pattern. Perhaps pick a class from progressbar.